cli/alsatian-cli.ts

```diff
@@ -1,18 +1,37 @@
 #! /usr/bin/env node
 
-import { createTestSet, TestRunner } from "../core/alsatian-core";
+import { createTestSet, TestRunner, TestOutput } from "../core/alsatian-core";
 import { CliTestRunner } from "./cli-test-runner";
 import { AlsatianCliOptions } from "./alsatian-cli-options";
 
+const {
```

Comment:
@jamesrichford No typings for `tap-bark` at the moment so it kicked off a bit. I can give it another go if you like.